src/components/UserPortal/UserSidebarOrg/UserSidebarOrg.tsx

```
@@ -175,9 +209,12 @@ const UserSidebarOrg = ({
             </button>
           )}
         </div> */}
-
         {/* Options List */}
-        <h5 className={styles.titleHeader}>{tCommon('menu')}</h5>
+        {!hideDrawer ? (
+          <h5 className={styles.titleHeader}>{tCommon('menu')}</h5>
+        ) : (
+          <div style={{ paddingBottom: '40px' }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)